geoportailv3/static/js/layerfactoryservice.js

```diff
@@ -134,6 +135,7 @@ app.getWmsLayer_ = function(ngeoDecorateLayer) {
     var imageExt = app.getImageExtension_(imageType);
     var layer = new ol.layer.Image({
       source: new ol.source.ImageWMS({
+        crossOrigin: 'anonymous',
```

Comment:
Because I thought it was part of a separate PR: https://github.com/Geoportail-Luxembourg/geoportailv3/pull/508/files.